BrowserKit/Sources/ComponentLibrary/Buttons/LinkButtonViewModel.swift

```diff
@@ -14,21 +15,21 @@ public struct LinkButtonViewModel {
 
     public let title: String
     public let a11yIdentifier: String
-    public let fontSize: CGFloat
+    public let font: TextStyling?
```

Comment:
Yes, I think the font needs to be nil because there are parts in the app where it's not called and initially it had a default value of 16 and now it doesn't have a default value set

Reply:
Ah in that case, lets use a default font style that matches the size 16 `callout` instead of nil